blackjack.js

```diff
@@ -1,15 +1,39 @@
-const blackjackDeck = getDeck();
+
+
+const blackjackDeck = () =>{
+
+const deck = getDeck();
+
+let val = deck.val;
+let displayVal = getDeck().displayVal;
```

Comment:
2 things about this . . . I can see that this `displayVal` variable is not being used anywhere in the program (VSCode kinda greys it out a little when that happens). So it can just be deleted. That will have a second effect of stopping the error, `Uncaught ReferenceError: displayVal is not defined` . . . because `getDeck()` returns an array of card objects, and has no `displaVal` property itself, that's only on the individual cards within.